assets/src/containers/AssignmentPlanningV2.js

```diff
@@ -46,6 +46,24 @@ const styles = theme => ({
   },
   mainProgressBar: {
     marginBottom: '50px'
+  },
+  graded: {
+    width: '10px',
+    height: '10px',
+    background: theme.palette.secondary.main,
+    display: 'inline-block'
+  },
+  ungraded: {
+    width: '10px',
+    height: '10px',
+    background: theme.palette.info.main,
+    display: 'inline-block'
+  },
+  unsubmitted: {
+    width: '10px',
+    height: '10px',
+    background: theme.palette.negative.main,
+    display: 'inline-block'
```

Comment:
The color schema was bar is changed more darkish grey from what I see in master, any reason for that?

Reply:
Not sure.

The definition in assets/src/defaultPalette.js has changed since May 8, 2020:

  /* The 'negative' color is used currently to indicate a negative, un-completed, un-viewed, or un-selected
     state. This value was arrived at through research and design work; use caution when modifying. For now,
     this should be used directly through theme.palette.negative.main instead of the color prop
     (i.e. color='negative'), which isn't fully supported. */
  negative: {
    main: '#B5B5B5'
  },

Reply:
I see currently this is the background color `#f1f1f1;` applied so as a result of that I have the bar color as this but your PR showed more darkish grey

![Screen Shot 2020-12-02 at 11 30 48 AM](https://user-images.githubusercontent.com/8579775/100902108-92575a80-3492-11eb-92ab-13089b35fccd.png)

Reply:
True. 

It is the palette.negative.main color, #B5B5B5. It has been applied to both Assignment Planning v1 and v2.